File: C++Slide02/Lab02/Ex02/TestEx02.cpp
// File: Lab02/Ex02/TestEx02.cpp
#include <iostream>
#include "RealCar.hpp"
#include "../../Lab01/Ex02/Driver.hpp" 


int main() {
    // Creating a Driver instance
    Driver driver("Bob", 35, 7);

    // Creating a RealCar instance
    RealCar realCar("DDD-4444", driver);

    // Interacting with RealCar
    realCar.turnOn();
    realCar.accelerate();
    realCar.honk();
    realCar.brake(15);
    realCar.turnOff();

    return 0;
}

File: ppLabs/Lab02/src/Lab02/Ex02/TestEx02.java
package Lab02.Ex02;

import Lab01.Ex02.Condutor;

public class TestEx02 {
    public static void main(String[] args) {

        Condutor driver = new Condutor("Chris", 20, 10);
        CarroReal car = new CarroReal("123-ADC", driver);

        car.connect();
        car.accelerate();
        car.honk();
        car.brake(10);
        car.disconnect();

    }
}
